client/verta/verta/_pipeline_orchestrator/_step_handler.py

```diff
@@ -48,51 +47,6 @@ def predecessors(self) -> Set[str]:
         return self._predecessors
 
 
-class ModelContainerStepHandler(_StepHandlerBase):
```

Comment:
This is moved to the orchestrator service (https://github.com/VertaAI/python-services/pull/222).